.changes/v0.14.0.md

```diff
@@ -0,0 +1,21 @@
+## v0.14.0 - 2024-11-08
+
+
+### 🔥 Breaking Changes
+- Rename `dagger-engine` core GraphQL query to `engine` by @Marcos Lilljedahl in https://github.com/dagger/dagger/pull/8568
```

Comment:
Oops, just realized this is gonna be a random Marcos 😄, will fix to be correct gh username